packages/wrangler/src/__tests__/sentry.test.ts

```diff
@@ -428,6 +453,308 @@ describe("sentry", () => {
+											pre_context: [],
+										},
+										{
+											colno: 0,
+											context_line: "",
+											filename: expect.any(String),
+											function: "",
+											in_app: false,
+											lineno: 0,
+											module: expect.any(String),
+											post_context: [],
+											pre_context: [],
+										},
+										{
+											colno: 0,
+											context_line: "",
+											filename: expect.any(String),
+											function: "",
+											in_app: false,
+											lineno: 0,
+											module: expect.any(String),
+											post_context: [],
+											pre_context: [],
+										},
+										{
+											colno: 0,
+											context_line: "",
+											filename: expect.any(String),
+											function: "",
+											in_app: false,
+											lineno: 0,
+											module: expect.any(String),
+											post_context: [],
+											pre_context: [],
+										},
+										{
+											colno: 0,
+											context_line: "",
+											filename: expect.any(String),
+											function: "",
+											in_app: false,
+											lineno: 0,
+											module: expect.any(String),
+											post_context: [],
+											pre_context: [],
+										},
+										{
+											colno: 0,
+											context_line: "",
+											filename: expect.any(String),
+											function: "",
+											in_app: false,
+											lineno: 0,
+											module: expect.any(String),
+											post_context: [],
+											pre_context: [],
+										},
+										{
+											colno: 0,
+											context_line: "",
+											filename: "/project/...",
+											function: "",
+											in_app: false,
+											lineno: 0,
+											module:
+												"@mswjs.interceptors.src.interceptors.fetch:index.ts",
+											post_context: [],
+											pre_context: [],
+										},
+										{
+											colno: 0,
+											context_line: "",
+											filename: "/project/...",
+											function: "",
+											in_app: false,
+											lineno: 0,
+											module:
+												"@mswjs.interceptors.src.interceptors.fetch:index.ts",
+											post_context: [],
+											pre_context: [],
+										},
+									],
+								},
+								type: "TypeError",
+								value: "Failed to fetch",
+							},
+						],
+					},
+					modules: {},
+					platform: "node",
+					release: "",
+					sdk: {
+						integrations: [
+							"InboundFilters",
+							"FunctionToString",
+							"LinkedErrors",
+							"Console",
+							"OnUncaughtException",
+							"OnUnhandledRejection",
+							"ContextLines",
+							"Context",
+							"Modules",
+						],
+						name: "sentry.javascript.node",
+						packages: [
+							{
+								name: "npm:@sentry/node",
+								version: "7.87.0",
+							},
+						],
+						version: "7.87.0",
+					},
+					timestamp: 0,
+				},
+				header: {
+					event_id: "",
+					sdk: {
+						name: "sentry.javascript.node",
+						version: "7.87.0",
```

Comment:
This will require us to update the test when ever we update the sentry client. Would it be better to match it with `expect.any(String)` instead?

There are a few more version number above.